### DISCUSSION TITLE
Multiple Release Schedules: Version one package, ignore the rest

### DISCUSSION THREAD

**Discussion by DavidVaness** [changesets/changesets#1328] [Q&A]
Hey, How do I handle Multiple Release Schedules?
I have a Monorepo and one package that I need to version and create the changelog.md for individually

there is the ignore flag that would allow me to ignore packages
```bash
changeset version --ignore pkg-a
```
Its very cumbersome to ignore 30+ apps/packages here. Is there a better way to handle this (regex?) 
For the rest of the releases, the versioning is alright because we deploy them together

My example: We have a storybook instance and I want to publish it independently from the rest of the app 

Should I move this to its own GH repo to avoid this?

**Comment by thanix-k** [comment]
I also have the same kind of use-case and haven't found any solutions yet.

We have a similar setup with multiple packages in a monorepo. Some packages need frequent independent releases. 
We want to release specific packages separately instead of always batching everything together.
